### DISCUSSION TITLE
Navigation menu with single js list

### DISCUSSION THREAD

**Discussion by thefubon** [alpinejs/alpine#1338] [1. Help]
Please tell me how to make a navigation menu with a separate list of links

![Frame 1](https://user-images.githubusercontent.com/43171309/119509472-57671d00-bd79-11eb-80bb-0043442767e6.png)

**Comment by gwleuverink** [comment]
You can use [x-for](https://github.com/alpinejs/alpine#x-for) 

If you use a server side templating engine (like blade for laravel) you could opt to use that for rendering those on the page. What suits you best depends on the problem & the stack you use.

**Reply by thefubon** [reply]
I cannot figure out how to do this. The documentation does not help me (

I have normal HTML and there are 5 pages in it and I want to make a menu but make a list separately so that I can edit one file and the menu changed on all HTML pages

**Reply by gwleuverink** [reply]
Do you get any errors when you copy this example? https://codepen.io/gwleuverink/pen/XWMgNjr

**Comment by thefubon** [comment]
https://codepen.io/KevinBatdorf/pen/PoZeBEv   -  About ak here - Only in the video navigation menu
